mpcs/src/basefold.rs

```diff
@@ -259,133 +266,87 @@ where
     /// Open a batch of polynomial commitments at several points.
     fn batch_open(
         pp: &Self::ProverParam,
-        num_instances: &[(usize, usize)],
-        fixed_comms: Option<&Self::CommitmentWithWitness>,
-        witin_comms: &Self::CommitmentWithWitness,
-        points: &[Point<E>],
-        // TODO this is only for debug purpose
-        evals: &[Vec<E>],
-        circuit_num_polys: &[(usize, usize)],
+        rounds: Vec<(
+            &Self::CommitmentWithWitness,
+            // for each matrix open at one point
+            Vec<(Point<E>, Vec<E>)>,
```

Comment:
comment seems mismatch with type definition for multiple points. how about simplify type to one point to serve current use case